scripts/cnode-helper-scripts/startupLogMonitor.sh

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/env bash
+#shellcheck disable=SC2086,SC2001,SC2154
+#shellcheck source=/dev/null
+
+# STARTUPLOG_DIR="/opt/cardano/cnode/guild-db/startuplog"
+# STARTUPLOG_DB="${STARTUPLOG_DIR}/startuplog.db"
+#logfile="/opt/cardano/cnode/logs/node0.json"
+
+######################################
+# Do NOT modify code below           #
+######################################
+
+createStartuplogDB() {
+  if ! mkdir -p "${STARTUPLOG_DIR}" 2>/dev/null; then echo "ERROR: failed to create directory to store blocklog: ${STARTUPLOG_DIR}" && return 1; fi
+  if [[ ! -f ${STARTUPLOG_DB} ]]; then # create a fresh DB with latest schema
+    sqlite3 ${STARTUPLOG_DB} <<-EOF
+			CREATE TABLE validationlog (id INTEGER PRIMARY KEY AUTOINCREMENT, event TEXT NOT NULL, at TEXT NOT NULL UNIQUE, env TEXT NOT NULL, final_chunk INTEGER, initial_chunk INTEGER);
+			CREATE TABLE replaylog (id INTEGER PRIMARY KEY AUTOINCREMENT, event TEXT NOT NULL, at TEXT NOT NULL UNIQUE, env TEXT NOT NULL, slot INTEGER, tip INTEGER);
+      CREATE TABLE statistics (id INTEGER PRIMARY KEY AUTOINCREMENT, event TEXT NOT NULL, start INTEGER, end INTEGER, env TEXT NOT NULL);
+			EOF
+    echo "SQLite startuplog DB created: ${STARTUPLOG_DB}"
+  fi
+}
+
+
+PARENT="$(dirname $0)"
+if [[ ! -f "${PARENT}"/env ]]; then
+    echo -e "\nCommon env file missing: ${PARENT}/env"
+    echo -e "This is a mandatory prerequisite, please install with guild-deploy.sh or manually download from GitHub\n"
+    exit 1
+fi
+
+if ! . "${PARENT}"/env offline ; then exit 1; fi
+
+[[ -z "${STARTUPLOG_DIR}" ]] && export STARTUPLOG_DIR="/opt/cardano/cnode/guild-db/startuplog"
+[[ -z "${STARTUPLOG_DB}" ]] && export STARTUPLOG_DB="${STARTUPLOG_DIR}/startuplog.db"
+
+#######################################################
+# Version Check                                       #
+#######################################################
+clear
+
+if [[ ${UPDATE_CHECK} = Y && ${SKIP_UPDATE} != Y ]]; then
+
+  echo "Checking for script updates..."
+
+  # Check availability of checkUpdate function
+  if [[ ! $(command -v checkUpdate) ]]; then
+    echo -e "\nCould not find checkUpdate function in env, make sure you're using official guild docos for installation!"
+    exit 1
+  fi
+
+  # check for env update
+  ENV_UPDATED=${BATCH_AUTO_UPDATE}
```

Comment:
BATCH_AUTO_UPDATE never set, please see topologyUpdater.sh for this one. Its used to initialize ENV_UPDATED and later as a second argument to silently update without user interaction.